### ISSUE TITLE
Raspberry Pi cannot handle 1872 x 1168 full screen map

### ISSUE THREAD

**Issue by alexandaday** [pyland/pyland#280]
When loading a map with screen resolution 1872 x 1168, and the PyScripter hidden (so the map is full screen), the game crashes.

This can be resolved by allocating 128mB for graphics opposed to 64mB for the Pi.

**Comment by Moosichu** [comment]
Well, the game should be running at a fixed resolution anyway. Because if the raspberry pi is connected to a 4K display in the extreme example, this will cause problems regardless of whether the PyScripter is hidden or not.

**Comment by alexandaday** [comment]
Perhaps we can set a maximum QT window size on the Pis as a temporary measure.

**Comment by asb** [comment]
The Pi doesn't support 4K display output, but this could certainly be an issue on other platforms. I'm not convinced a fixed resolution is the answer to all our problems here (scaling can look ugly). Though it certainly would make design of levels like the crocodiles level easier (this level becomes very difficult on a small screen as you can't see across the river).

**Comment by Moosichu** [comment]
Yeah, regardless I think the game should support resolutions decoupled from the monitor one anyway. Especially if in 10 years time people are playing it on their 12inch 16K laptops :stuck_out_tongue: Even if we let the user set it arbitrarily, because once we can support that we can do what we want then with it.
